packages/workbox-build/src/lib/populate-sw-template.js

```diff
@@ -47,22 +44,27 @@ module.exports = ({
       undefined,
   };
 
-  let workboxOptionsString = JSON.stringify(workboxOptions, null, 2);
+  let precacheOptionsString = JSON.stringify(precacheOptions);
```

Comment:
Personally I liked the indented version, especially as it meant it was more human readable.